packages/story-editor/src/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -81,7 +88,11 @@ const getVideoResource = async (file) => {
 
   await seekVideo(videoEl);
   const hasAudio = hasVideoGotAudio(videoEl);
-  const posterFile = await getImageFromVideo(videoEl);
+  const posterFile = blobToFile(
+    await getImageFromVideo(videoEl),
+    getPosterName(getFileName(file)),
+    MEDIA_POSTER_IMAGE_MIME_TYPE
+  );
```

Comment:
Why is this change needed?

Reply:
It simplified the `posterFile` handling in the media upload queue which manually set a poster file name here:

https://github.com/google/web-stories-wp/blob/417c819d3c12adaf5e33a6c29f59c2050fabeaeb/packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js#L280

This way now we know that `posterFile` always has a proper name that we can use.